src/Pretzel.Logic/Templating/Jekyll/SiteContextDrop.cs

```diff
@@ -37,8 +43,27 @@ public Hash ToHash()
             x["tags"] = context.Tags;
             x["categories"] = context.Categories;
             x["time"] = Time;
+            x["data"] = Data;
 
             return x;
         }
+
+        private void FillData(string dataPath)
```

Comment:
It is preferable to fill the data in `SiteContextGenerator` and use `IFileSystem` to get all files in order to be able to do some unit tests.